src/util.cpp

```diff
@@ -682,6 +682,17 @@ void ArgsManager::ParseParameters(int argc, const char* const argv[])
             m_override_args[key].push_back(val);
         }
     }
+
+    // we do not allow -includeconf from command line, so we clear it here
+    auto it = m_override_args.find("-includeconf");
+    if (it != m_override_args.end()) {
+        if (it->second.size() > 0) {
```

Comment:
It'd be nice if we could be more aggressive here and fail to start if a `-includeconf` argument is provided on the command line (since continuing to startup would be continuing with config that the user has specified that they don't want).

`ParseParameters()` doesn't currently have a way to return errors, and callers don't wrap calls with try-except (unlike `GetChainName()` for example). Perhaps a future PR could change the function to return a bool, so we could indicate an error to the caller and abort startup.

Reply:
Makes sense. I'm leaving it for now as it sounds like a future thing, I think.